helpers-assets/templates/translation-discussion-comments.php

```diff
@@ -5,21 +5,22 @@
 ?>
 <div class="discussion-wrapper">
 	<?php if ( $number = count( $comments ) ) : ?>
-		<h6><?php printf( _n( '%s Comment', '%s Comments', $number ), number_format_i18n( $number ) ); ?>
-		<?php if ( $locale_slug ) : ?>
-			(<?php echo esc_html( $locale_slug ); ?>)
-			<?php
-			$countLocaleComments = 0;
-			foreach ( $comments as $comment ) {
-				$comment_locale = get_comment_meta( $comment->comment_ID, 'locale', true );
-				if ( $locale_slug == $comment_locale ) {
-					$countLocaleComments++;
-				}
-			}
-			?>
-						
+		<?php if ( ! $locale_slug ) : ?>
```

Comment:
Is this still correct? Wouldn't it make more sense to check for `if ( $languages )` already here and swap the if contents?

Reply:
This is in place to check if user is on the English discussions page.